netlify.toml

```diff
@@ -28,3 +28,13 @@
 [[redirects]]
  from = "/public/api/vue2/*"
  to = "/api/vue2/"
+
+[[headers]]
+   for = "/*"
+   [headers.values]
+      Access-Control-Allow-Origin = "*"
+      Referrer-Policy = "strict-origin-when-cross-origin"
+      Content-Security-Policy = "default-src 'self'; script-src 'self' 'unsafe-eval' 'sha256-mC5lwOEBZZZXJoN3sDvzxnxAdNIEKujq9NSXgmhc4HM=' 'sha256-iIf+c/EMxKD/FXoUDy0YsZ3mE+JhzPsmR+aVbrjkdwM=' static.cloudflareinsights.com; style-src 'self' 'unsafe-inline'; img-src 'self' data:; object-src 'none'; frame-ancestors 'self';"
+      X-Frame-Options = "SAMEORIGIN"
```

Comment:
Shouldn't there be a placeholder here?

Reply:
Yes, but since the calculated hash was always wrong, I inserted the expected hash directly.